spec/features/trainer_spec.rb

```diff
@@ -41,8 +40,9 @@
 
   describe "training with one card" do
     before do
-      card = create(:card)
-      card.update_attribute(:review_date, Time.now)
+      user = create(:user_with_cards)
+      user.cards.each { |card| card.update_attribute(:review_date,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.